### PR TITLE
rg-167: Add Docker

### PR DESCRIPTION
**Pull request by RevolveStrm**
image
image
image
image
image
image
image

### CODE REVIEW

**Review by MaxPoplavskiy — APPROVED** [review]

File: backend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+WORKDIR /app/backend
+
+RUN npm install
+
+RUN npm run build
+
+#RUN npm run migrate:dev
+
+#RUN npm run seed:run
```
Comment (jurok3x) [inline backend/Dockerfile:36]: initialize db in docker-compose:
image

File: docker-compose.yml [hunk 2]
```diff
@@ -1,11 +1,62 @@
...
+    api:
+        container_name: 'api'
+        build:
+            context: ./
+            dockerfile: ./backend/Dockerfile
+        restart: unless-stopped
+        environment:
+            - NODE_ENV=production
+            - PORT=3001
+            - HOST=0.0.0.0
```
Comment (jurok3x) [inline docker-compose.yml:35]: is this host Url correct? Api and databaseare running from different containers. You should specify **db** as host.

**Review by jurok3x — CHANGES_REQUESTED** [review]

**Comment by vlad-blip** [comment]
[Add docker](https://trello.com/c/FNG4ezmM/167-add-docker)

File: docker-compose.yml (hunk 2 above)
Comment (RevolveStrm) [reply docker-compose.yml:35]: Yes, it is correct. I defined this HOST variable as a server host (api service). You can see that api service connects to DB through url in ENVs 

DB_CONNECTION_STRING=pg://postgres_user:postgres_password@**db**:5432/postgres_db - _here **db** is a host._

**Comment by RevolveStrm** [comment]
Temporary blocked!)

**Pull request closed without merging**